src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -113,4 +117,165 @@ public void execute_unknownCommandWord() throws Exception {
+            return new Task(
+                    new Name("Task " + seed)
+            );
+        }
+
+        /** Generates the correct add command based on the task given */
+        String generateAddCommand(Task p) {
+            StringBuffer cmd = new StringBuffer();
+
+            cmd.append("add ");
+
+            cmd.append(p.name.toString());
+
+            return cmd.toString();
+        }
+
+        /**
+         * Generates an TaskBook with auto-generated tasks.
+         */
+        TaskBook generateAddressBook(int numGenerated) throws Exception {
+            TaskBook taskBook = new TaskBook();
+            addToTaskBook(taskBook, numGenerated);
+            return taskBook;
+        }
+
+        /**
+         * Generates an TaskBook based on the list of Tasks given.
+         */
+        TaskBook generateAddressBook(List<Task> tasks) throws Exception {
+            TaskBook taskBook = new TaskBook();
+            addToTaskBook(taskBook, tasks);
+            return taskBook;
+        }
+
+        /**
+         * Adds auto-generated Task objects to the given TaskBook
+         * @param taskBook The TaskBook to which the Tasks will be added
+         */
+        void addToTaskBook(TaskBook taskBook, int numGenerated) throws Exception {
+            addToTaskBook(taskBook, generateTaskList(numGenerated));
+        }
+
+        /**
+         * Adds the given list of Tasks to the given TaskBook
+         */
+        void addToTaskBook(TaskBook taskBook, List<Task> tasksToAdd) throws Exception {
+            for (Task p: tasksToAdd) {
+                taskBook.addTask(p);
+            }
+        }
+
+        /**
+         * Adds auto-generated Task objects to the given model
+         * @param model The model to which the Tasks will be added
+         */
+        void addToModel(Model model, int numGenerated) throws Exception {
+            addToModel(model, generateTaskList(numGenerated));
+        }
+
+        /**
+         * Adds the given list of Tasks to the given model
+         */
+        void addToModel(Model model, List<Task> tasksToAdd) throws Exception {
+            for (Task p: tasksToAdd) {
+                model.addTask(p);
+            }
+        }
+
+        /**
+         * Generates a list of Tasks based on the flags.
+         */
+        List<Task> generateTaskList(int numGenerated) throws Exception {
+            List<Task> tasks = new ArrayList<>();
+            for (int i = 1; i <= numGenerated; i++) {
+                tasks.add(generateTask(i));
+            }
+            return tasks;
+        }
+
+        List<Task> generateTaskList(Task... tasks) {
+            return Arrays.asList(tasks);
+        }
+
+        /**
+         * Generates a Task object with given name. Other fields will have some dummy values.
+         */
+        Task generateTaskWithName(String name) throws Exception {
+            return new Task(
+                    new Name(name)
+            );
+        }
+    }
```

Comment:
Sorry, this comment was meant for this thread:

Maybe consider downloading the commit from #134 and building on top that instead?